### PR TITLE
az-aks: add page

### PR DESCRIPTION
**Pull request by Vishnudas2003**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by gutjuri — APPROVED** [review]
Thank you very much!

File: pages/common/az-aks.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# aks
```
Comment (acuteenvy) [inline pages/common/az-aks.md:1]:
```suggestion
# az aks
```

File: pages/common/az-aks.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+> The official CLI tool for Azure Kubernetes Service (AKS).
+> Manage Azure Kubernetes Service (AKS) clusters.
+> Official documentation: <https://docs.microsoft.com/cli/azure/aks>.
```
Comment (acuteenvy) [inline pages/common/az-aks.md:5]: https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#descriptions
```suggestion
> Manage Azure Kubernetes Service (AKS) clusters.
> More information: <https://docs.microsoft.com/cli/azure/aks>.
```

File: pages/common/az-aks.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# az aks
+
+> Manage Azure Kubernetes Service (AKS) clusters.
+> Part of `az`.
```
Comment (kbdharun) [inline pages/common/az-aks.md:4]:
```suggestion
> Part of `azure-cli`.
```

**Review by kbdharun — APPROVED** [review]
LGTM, Welcome to tldr and thanks for your contribution.

**Pull request merged**